.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -25,21 +24,8 @@
   - [ ] Not Applicable
   - **OR**
   - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
-        - [ ] Issue filed: <!-- Number/link of that issue here -->
+    - [ ] Issue filed: <!-- Number/link of that issue here -->
 - **Testing - New and feature**
   - [ ] N/A or can only be tested interactively
   - **OR**
   - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
-- **Tooling**
-  - [ ] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
-  - **OR**
-  - [ ] I have considered the user experience from a tooling perspective and opened an issue in the relevant tool repository. This may include:
-    - [ ] Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode
```

Comment:
This was actually my favorite section of the PR template 😅 but yeah if no one is reviewing it, it's pointless.

Reply:
The only way to check a tool is to run the tests if we add them.